page/partials/head.ejs

```diff
@@ -7,15 +7,17 @@
     <meta name="keywords" content="neufund, ico, neumark, venture capital, ethereum, startup, blockchain" />
     <meta name="robots" content="/robots.txt" />
 
-    <meta property="og:title" content="Neufund |  <%= title %>" />
+    <meta property="og:title" content="Neufund | <%= title %>" />
     <meta property="og:image" content="assets/social_logo.png" />
-    <meta property="og:url" content="http://neufund.org/" />
+    <meta property="og:url" content="https://commit.neufund.org" />
+    <meta property="og:type" content="website" />
+    <meta property="og:description" content="Neufund’s vision is to invite everyone to join the bootstrapping of a new two-sided marketplace and an investment ecosystem, and to once and for all democratise innovation and creation." />
```

Comment:
could you export description that is repeated few time in this file to content.yml?

Reply:
Good idea